src/docs/Components/Hero/previews/box.preview.html

```diff
@@ -0,0 +1,22 @@
+<div class="example">
```

Comment:
Since this is not going to be committed now, can you please remove this file from the commit?